### PR TITLE
Update spacing

### PR DESCRIPTION
**Pull request by hayleycd**
Adjusting line height in order to improve readability on the blog.

### CODE REVIEW

**Comment by hayleycd** [comment]
Partially addresses #1106

File: gcp/appengine/blog/layouts/posts/single.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.content {
```
Comment (oliverchang) [inline gcp/appengine/blog/layouts/posts/single.html:1]: This should go into our styles.scss instead. @another-rex can you please help Hayley with this change?

**Comment by another-rex** [comment]
We try to put all the css into styles.scss located here: [gcp/appengine/frontend3/src/styles.scss](https://github.com/google/osv.dev/blob/master/gcp/appengine/frontend3/src/styles.scss). 

At the bottom of that file you'll find the `.blog-post-page` section, should look something like this:
```
...

.blog-post-page {
  .post {
    ...
    p {
      margin-bottom: 16px;
// Add line height here
    }
  }
}
```
I think putting the line height value in `p` tag at the bottom would work.

I would suggest also putting it in the `p` tag under the `.blog-post` section just above:

```
...

.blog-page {
  .posts {
...
    .blog-entry {
...
      p {
        padding-bottom: 16px;
// another line height entry here
      }
    }
  }
}

.blog-post-page {
...
```

this will change the line height on the base /blog/ page where the blog summaries are listed.

**Comment by hayleycd** [comment]
@another-rex @oliverchang 

Thanks for pointing me in the right direction. I am going to close this, and will open another PR editing this file tomorrow. I might include a couple of other properties besides line-height.

**Comment by another-rex** [comment]
Also, just playing around with the values in inspect element, 1.8 line height seems pretty big, maybe 1.6 or 1.7 looks a bit better? 

I'm not sure what the guidelines around this is though.

**Pull request closed without merging**